Directory.Packages.props

```diff
@@ -123,6 +123,7 @@
     <PackageVersion Include="Swashbuckle.AspNetCore" Version="6.5.0" />
     <PackageVersion Include="Testcontainers.RabbitMq" Version="$(TestcontainersPackageVersion)" />
     <PackageVersion Include="Testcontainers.MongoDb" Version="$(TestcontainersPackageVersion)" />
+    <PackageVersion Include="Testcontainers.Redis" Version="$(TestcontainersPackageVersion)" />
```

Comment:
Do these always version together? Or are there cases were ones may get serviced but others don't?

Reply:
AFAICT, they seem to be versioned together.